modules/administration-guide/partials/proc_enabling-readiness-init-containers-for-olm-installer.adoc

```diff
@@ -0,0 +1,44 @@
+:_module-type: PROCEDURE
+
+[id="enabling-readiness-init-containers-for-olm-installer_{context}"]
+= Enabling readiness init containers for OLM installer
+
+To enable or disable readiness init containers for {prod-short} installed using the Operator Lifecycle Manager (OLM) installer:
+
+.Prerequisites
+
+* {prod-short} is installed with OLM installer.
+
+.Procedure
+
+. Find the {prod-short} Operator subscription name:
++
+[subs="+attributes,+quotes"]
+----
+$ {orch-cli} get subscriptions -n {prod-namespace}
+----
+
+. Find the CSV name in the {prod-short} Operator subscription:
++
+[source,shell,subs="+quotes,+attributes"]
+----
+$ {orch-cli} get subscription <subscription-name> -n {prod-namespace} -o yaml | grep installedCSV
+----
+
+. Edit the CSV:
```

Comment:
```suggestion
. Edit the `ClusterServiceVersion` (CSV) YAML manifest:
```
I hope CSV=ClusterServiceVersion is correct.

Reply:
Yes, it is